docs/source/using/profiles.adoc

```diff
@@ -31,10 +31,10 @@ There are two ways to create a new profile.
 
 [NOTE]
 ====
-Profile name can only consist of alphanumeric characters and *-* as a separator, for example:
+Profile name can only consist of alphanumeric characters (underscore *_* allowed) and *-* as a separator, for example:
 
-- foo, bar123, demo-profile and 1stProfile are considered as valid profile names.
-- $foo, bar#@, profile@123 are invalid profile names.
+- foo, bar_123, demo-profile and 1stProfile are considered as valid profile names.
+- $foo, bar#@, profile@123, -foo are invalid profile names.
```

Comment:
So why disallows '-foo'. I guess I understand now what you wanted to say in your explanation, but that is not clear at all. Effectively _foo would be ok, but -foo not?